tests/go.mod

```diff
@@ -0,0 +1,15 @@
+module chip-tool-snap-tests
+
+go 1.21.6
+
+require github.com/canonical/matter-snap-testing v0.0.0-20240115153232-0f3c7fc00176
+
+replace github.com/canonical/matter-snap-testing v0.0.0-20240115153232-0f3c7fc00176 => github.com/farshidtz/matter-snap-testing v0.0.0-20240122134356-4b20ea079b75
```

Comment:
Need to update once https://github.com/canonical/matter-snap-testing/pull/8 has been merged.

Reply:
https://github.com/canonical/chip-tool-snap/pull/36/commits/fca276aab3aaed82587544ee87d3e3ba020acef5